mecanum_drive_controller/src/mecanum_drive_controller.yaml

```diff
@@ -0,0 +1,100 @@
+mecanum_drive_controller:
+  reference_timeout: {
+    type: double,
+    default_value: 0.0,
+    description: "Timeout for controller references after which they will be reset. This is especially useful for controllers that can cause unwanted and dangerous behaviour if reference is not reset, e.g., velocity controllers. If value is 0 the reference is reset after each run.",
+
+  }
+  wheel0_name: {
+    type: string,
+    default_value: "front_left_wheel_joint",
```

Comment:
I would also delete wheel names' default values because they should be always explicitly set.

Reply:
they will always be same for 4WD right,  may be we can create a array wheels_names, and shouldn't default value always be there in case no value is set by the user?

Reply:
No the names are never the same in robot, or at least this is an assumption that may not be made.